package.json

```diff
@@ -13,11 +13,12 @@
   "scripts": {
     "build": "npm run clean && tsc",
     "clean": "npx rimraf dist",
+    "format": "prettier --write \"**/*.ts\" && eslint --fix --ext .ts,.js,.json .",
+    "lint": "prettier --check \"**/*.ts\"  && eslint --ext .ts,.js,.json .",
+    "prepare": "husky install",
```

Comment:
Shouldn't this be in a post-install script? Otherwise mention it in the readme please if this is a required setup step.